src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -644,7 +644,7 @@ Subgraph::ControlFlowPasses Subgraph::getControlFlowPasses() {
                                            ov::intel_cpu::pass::InsertBrgemmCopyBuffers);
     SNIPPETS_REGISTER_PASS_ABSOLUTE_X86_64(Place::PipelineEnd,
                                            ov::intel_cpu::pass::InitRepackedConstantInputs,
-                                           context->getParamsCache(),
+                                           context->getSnippetsParamsCache(),
```

Comment:
@a-sidorova please note that in your PR https://github.com/openvinotoolkit/openvino/pull/31024 we should pass `getParamsCache` to `ov::intel_cpu::pass::RepackMatMulWeights` for `node::Reorder::reorderData()` helper